docs/design/operator.md

```diff
@@ -0,0 +1,200 @@
+# Gluster Operator 1.0 Design
```

Comment:
Not sure if 1.0 is really appropriate here. It sounds premature and might set false expectations.